common/djangoapps/entitlements/models.py

```diff
@@ -452,3 +452,15 @@ def __unicode__(self):
             self.support_user,
             self.reason,
         )
+
+    @classmethod
+    def get_support_reasons_list(cls):
+        """
+        Method for retrieving a serializable version of the entitlement support reasons
+
+        Returns Array of reasons
+        """
+        reasons_list = []
+        for value, label in cls.ENTITLEMENT_SUPPORT_REASONS:
```

Comment:
use a map here?

```python
return [
  {'value': value, 'label': label} for value, label 
  in cls.ENTITLEMENT_SUPPORT_REASONS
]
```

tested like this

![image](https://user-images.githubusercontent.com/8136030/36922297-55ee7982-1e35-11e8-9cd0-b03de6167f02.png)